extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -96,13 +110,45 @@ export class NetCoreTool {
 		return undefined;
 	}
 
+	private get isNetCoreVersionSupported(): boolean | undefined {
+		try {
+			let spawnSync = require('child_process').spawnSync;
```

Comment:
Requires should go at the top of the file and use the import syntax `import * as child_process from 'child_process'`